spec/controllers/customers.spec.js

```diff
@@ -0,0 +1,108 @@
+var request = require('request')
+var base_url = "http://localhost:3000/customers"
+
+
+describe("#index", function() {
+    it("returns a Success response", function(done) {
+      request.get("http://localhost:3000/customers", function(error, response, body) {
+        expect(response.statusCode).toBe(200)
+        done()
+      })
+    })
+
+
+    it("should be json", function(done) {
+        request.get("http://localhost:3000/customers", function(error, response, body) {
+          expect(response.headers['content-type']).toContain('application/json')
+          done()
+        })
+      })
+      it("should be an array of objects", function(done) {
+      request.get("http://localhost:3000/customers", function(error, response, body) {
+        var customers = JSON.parse(body)
+        expect(typeof customers).toEqual('object')
+        done()
```

Comment:
Your test description states that this should be an array, but it looks like you are actually verifying that it is an object.